data/discord_messages.yaml

```
@@ -1662,3 +1418,21 @@ bp:
 
 
     If that's not working out, this is a good secondary spot for help.
+python:
+  description: "Python"
+  content: >-
+    Python 3.9 is the minimum version of Python supported by Home Assistant,
+    with 3.9 being recommended. If you're using a Python virtual environment
+    install you can install an updated version of Python, then manually upgrade
+    your venv as described in
+    <https://blog.ceard.tech/2017/12/upgrading-python-virtual-environment.html>
+check:
+  description: "Check configuration"
```

```suggestion
  description: "Check configuration with CLI"
```
Alternative description 🤷‍♂️

---

Let's not add this, and we add the UI check in the future once it's on par.

---

https://github.com/home-assistant/service-hub/pull/82#discussion_r988377125

---

I meant your change of "with CLI"